### PR TITLE
Add fields to store planning time in pick-and-place

### PR DESCRIPTION
**Pull request by aochiai**
Add new fields to store planning time of the pick-and-place capability.
`MoveGroup.action` already has this.

### CODE REVIEW

**Comment by davetcoleman** [comment]
This will break ABI in a difficult way for Kinetic. I believe this should be merged into ROS-M, which we haven't branched for yet. We are having a maintainer meeting on March 20th and will be discussing branching. Would you like to join this discussion?

**Comment by aochiai** [comment]
It makes sense to me to put this off until ROS-M.
I'm interested in the maintainer meeting. (Mostly out of curiosity)
Please let me know when you find out the holding time.

**Comment by v4hn** [comment]
I agree, we should not merge this into kinetic.
Let's discuss more in the meeting.

**Comment by davetcoleman** [comment]
@aochiai please email me so I can invite you. 
dave@picknik.ai

**Comment by davetcoleman** [comment]
@aochiai I have created the new branch

File: action/Pickup.action [hunk 1]
```diff
@@ -65,6 +65,8 @@ string[] trajectory_descriptions
 # The performed grasp, if attempt was successful
 Grasp grasp
 
+# The amount of time in seconds it took to complete the plan
```
Comment (mlautman) [inline action/Pickup.action]: This seems ambiguous. Is it the time to plan or the time to execute the plan? Maybe reword to "it took to generate a plan"

File: action/Place.action [hunk 2]
```diff
@@ -56,6 +56,8 @@ string[] trajectory_descriptions
 # The successful place location, if any
 PlaceLocation place_location
 
+# The amount of time in seconds it took to complete the plan
```
Comment (mlautman) [inline action/Place.action]: This seems ambiguous. Is it the time to plan or the time to execute the plan? Maybe reword to "it took to generate a plan"

**Comment by mlautman** [comment]
@aochiai  Can you rewrite the comment to make it less ambiguous in another PR?
